apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -508,17 +517,10 @@ export function AppRoot({
         let ballotContextProviderChild = <Ballot />;
         // Pages that condition on state machine state aren't nested under Ballot because Ballot uses
         // frontend browser routing for flow control and is completely independent of the state machine.
-        // We still want to nest pages that condition on the state machine under BallotContext so we render them here.
+        // We still want to nest some pages that condition on the state machine under BallotContext so we render them here.
         if (stateMachineState === 'presenting_ballot') {
           ballotContextProviderChild = <ValidateBallotPage />;
         }
-        if (
-          stateMachineState === 'waiting_for_invalidated_ballot_confirmation'
-        ) {
-          ballotContextProviderChild = (
-            <BallotInvalidatedPage authStatus={authStatus} />
-          );
-        }
```

Comment:
Since we are no longer restarting the voting session, we don't need access to `BallotContext`